data/css/modules/card/_media.scss

```diff
@@ -2,23 +2,40 @@
     background-color: white;
 
     &__caption {
-        font-size: 20px;
+        font-size: 16px;
         font-family: $body-font;
-        color: #666666;
-        padding: 20px 20px 0px 20px;
+        color: transparentize(black, 1 - 0.60);
     }
 
     &__attributionText {
         font-weight: bold;
-        font-size: 16px;
-        font-family: $display-font;
+        font-size: 14px;
+        font-family: $support-font;
         color: $accent-light-color;
-        padding: 10px 20px 20px 20px;
 
         &:hover {
             color: $accent-dark-color;
         }
     }
+
+    &__caption,
+    &__attributionText {
+        padding: 25px;
+    }
+}
+
+// When both the caption & attribution are present in the lightbox,
+// edit padding to accommodate both
+.CardMedia--caption.CardMedia--attribution {
+    .CardMedia {
```

Comment:
This doesn't seem correct; this would select a `Card.Media` module _inside_ another `Card.Media` module with caption and attribution. It looks like it should be `.CardMedia.CardMedia--caption.CardMedia--attribution` or else nested the other way around.